src/common/components/copyButton/CopyButton.tsx

```diff
@@ -0,0 +1,67 @@
+import classNames from "classnames";
+import copy from "copy-to-clipboard";
+import React from "react";
+
+const MESSAGE_DISPLAY_TIME = 4000; // 4s
+
+function useResetState<S>(
```

Comment:
This function could be moved to separate file e.g. /src/hooks/useResetState.tsx

Reply:
Done!